File: Loenn/triggers/EntityReskinTrigger.lua
local entityReskinTrigger = {}

entityReskinTrigger.name = "SkinModHelper/EntityReskinTrigger"
entityReskinTrigger.placements = {
    {
        name = "normal",
        data = {
            width = 32,
            height = 40,
			entityIndex = -1,
            entityFullName = "Celeste.Strawberry",
			
			oneUse = true,
			newSpriteID = "strawberry_",
        }
    }
}

entityReskinTrigger.fieldInformation = {
    entityIndex = {
        fieldType = "integer",
        minimumValue = -1,
    }
}

return entityReskinTrigger

File: Ahorn/triggers/EntityReskinTrigger.jl
module SkinModHelperEntityReskinTrigger
using ..Ahorn, Maple

@mapdef Trigger "SkinModHelper/EntityReskinTrigger" EntityReskinTrigger(x::Integer, y::Integer, width::Integer=32, height::Integer=40,
	entityIndex::Integer=-1, entityFullName::String="Celeste.Strawberry", 
	oneUse::Bool=true, newSpriteID::String="strawberry_")

const placements = Ahorn.PlacementDict(
	"Entity Reskin Trigger (Skin Mod Helper)" => Ahorn.EntityPlacement(
		EntityReskinTrigger,
		"rectangle"
	)
)

end